conf/locale/ru/LC_MESSAGES/djangojs.po

```diff
@@ -8412,7 +8417,11 @@ msgstr "или создать новую в этом сервисе"
 
 #: lms/templates/student_account/register.underscore
 msgid "Create account"
-msgstr ""
+msgstr "Coздать профиль"
+
+#: lms/templates/student_account/register.underscore
+msgid "Already have an {platformName} account?"
```

Comment:
@RuslanSkiraRaccoongang please push respective changes in the`register.underscore` with a new string marked for translation

Reply:
Also, let's probably not wait for the platform config to arrive.
Once any changes happen in strings, I'll update po files

Reply:
![image](https://user-images.githubusercontent.com/48920645/58712998-5f7a6500-83ca-11e9-8520-fe9b7b1876a6.png)

Reply:
Need to always add this data